frontend/src/app/testQueue/QueueItem.jsx

```diff
@@ -60,24 +60,33 @@ const UPDATE_AOE = gql`
 `;
 
 const AreYouSure = ({ patientName, cancelHandler, continueHandler }) => (
-  <CMSDialog
-    onExit={cancelHandler}
-    heading="You have incomplete data"
-    size="narrow"
-    alert={true}
-    actions={
-      <React.Fragment>
-        <Button onClick={cancelHandler} variation="transparent">
-          No, go back
-        </Button>
-        <Button onClick={continueHandler}>Submit Anyways</Button>
-      </React.Fragment>
-    }
+  <Modal
+    isOpen={true}
+    style={{
+      content: {
+        top: "50%",
+        left: "50%",
+        width: "40%",
+        marginRight: "-50%",
+        transform: "translate(-50%, -50%)",
+      },
+    }}
```

Comment:
would be nice if we eventually the styling out of the component, but no biggie for now